src/commands/sbom/language.ts

```diff
@@ -15,6 +15,15 @@ export const getLanguageFromComponent = (component: any): DependencyLanguage | u
     if (component['purl'].includes('pkg:cargo')) {
       return DependencyLanguage.RUST
     }
+    if (component['purl'].includes('pkg:gem')) {
+      return DependencyLanguage.RUBY
+    }
+    if (component['purl'].includes('pkg:maven')) {
+      return DependencyLanguage.RUBY
+    }
```

Comment:
Is `pkg:maven` strictly equal to Ruby in our tool? Isn't Maven used for different language like Java or Scala also?

Reply:
Thanks for catching this. This should be JVM.